angular-client/src/components/history/carousel.component.html

```diff
@@ -0,0 +1,21 @@
+<p-carousel [value]="runs" [numVisible]="1" [numScroll]="1" [circular]="true" [showIndicators]="false">
+  <ng-template let-run pTemplate="item">
```

Comment:
why do you have this template thing

Reply:
in the docs for the carousel component it uses these ng-templates for its header, content, footer etc. This one specifically allows a run to be displayed as an item of the carousel